crates/derive/src/online/alloy_providers.rs

```diff
@@ -72,8 +69,9 @@ impl ChainProvider for AlloyChainProvider {
             return Ok(header.clone());
         }
 
-        let raw_header: Bytes =
-            self.inner.request("debug_getRawHeader", [hash]).await.map_err(|e| anyhow!(e))?;
+        let method = alloc::borrow::Cow::Borrowed("debug_getRawHeader");
```

Comment:
Can we just `.into()` on `"<method>"` rather than doing the long instantiation on all of these? the clone on write object has a direct `From<&'static str>` impl [here](https://doc.rust-lang.org/stable/std/borrow/enum.Cow.html#impl-From%3C%26str%3E-for-Cow%3C'a,+str%3E)